### PR TITLE
Add iPad support

### PR DESCRIPTION
**Pull request by kylef**
ios simulator screen shot 8 jul 2015 10 28 10

### CODE REVIEW

File: fastlane/Snapfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ devices([
   'iPhone 6 Plus',
   'iPhone 5',
   'iPhone 4s',
+  'iPad Air',
```
Comment (danielgtaylor) [inline fastlane/Snapfile]: Should this also be listing other iPad versions?

**Comment by danielgtaylor** [comment]
Just had the one question above. I think this looks good otherwise.

File: fastlane/Snapfile (hunk 1 above)
Comment (kylef) [inline fastlane/Snapfile]: All of the iPads use the same screen resolution so we don't need to take any others.

Comment (danielgtaylor) [inline fastlane/Snapfile]: Ah okay good to know, thanks!

**Pull request merged**